tests/test_commit_message.py

```diff
@@ -36,6 +36,58 @@ def test_commit_message_reviewers_replaced():
     assert commit_message == (FIRST_LINE, COMMIT_MESSAGE)
 
 
+@pytest.mark.parametrize(
+    "reviewers",
+    [
+        ["a"],
+        ["aa"],
+        [".a"],
+        ["..a"],
+        ["...a"],
+        ["a...a"],
+        ["a.b"],
+        ["a.b.c"],
+        ["-.-.-"],
+        ["a", "aa", ".a", "..a", "...a", "a...a", "a.b", "a.b.c", "-.-.-"],
+    ],
+)
+def test_various_reviewer_name_combinations(reviewers):
```

Comment:
Since we don't actually parse any of these, and just write them out, this doesn't add much to our test coverage. Maybe you could re-work this test to mimic `test_group_reviewers_replaced`, but focused on our period parsing?

Reply:
Noted and moved a subset of these to `test_group_reviewers_replaced`.